src/api/app/models/package.rb

```diff
@@ -933,9 +933,11 @@ def jobhistory_list(project, repository, arch, options = {})
     options[:limit] = 100 if options[:limit].blank?
     options[:package] = name if options[:package].blank?
 
-    results = Jobhistory.find_hashed(project: project.name, package: options[:package],
-                                     repository: repository, arch: arch,
-                                     limit: options[:limit])
+    begin
+      results = Xmlhash.parse(Backend::Api::BuildResults::JobHistory.all_for_package(project.name, options[:package], repository, arch, options[:limit]))
```

Comment:
I don't think collapsing it is better... too long to read it... :unamused:

Reply:
there is no rubocop about it - so please stop nitpicking

Reply:
this is not related with Rubocop, Rubocop complains about line lenght, but it doesn't know about readability. This is in my opinion difficult to read. :unamused:

Reply:
then fix the other 23 places first!

Reply:
you have to start somewhere...